testing-plugin/build.gradle

```diff
@@ -15,11 +16,12 @@ dependencies {
 
     compileOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     compileOnly "io.github.classgraph:classgraph:$class_graph_version"
-    implementation "org.assertj:assertj-core:$assertj_version"
+    compile "org.assertj:assertj-core:$assertj_version"
 
     // TODO: temporary dependency until PR is approved and published
     // implementation "com.github.tschuchortdev:kotlin-compile-testing:$kotlin_compile_testing"
-    implementation "com.github.arrow-kt:kotlin-compile-testing:$kotlin_compile_testing"
+    // implementation "com.github.arrow-kt:kotlin-compile-testing:$kotlin_compile_testing"
+    compile "com.github.tschuchortdev:kotlin-compile-testing:1.2.4-SNAPSHOT"
```

Comment:
This version doesn't exists: https://github.com/tschuchortdev/kotlin-compile-testing/releases
I think the only option would be updating `$kotlin_compile_testing` to `1.3.0` with the previous dependency: https://github.com/arrow-kt/kotlin-compile-testing/releases

And it's also used in runtime, so `compile` will raise errors.

Reply:
It's a snapshot release so it's not listed under the regular releases. I was going to wait with the official 1.2.4 release until the PR for easier compiler plugin support is merged.